aravisGigEApp/src/aravisCamera.cpp

```diff
@@ -625,15 +665,18 @@ asynStatus aravisCamera::writeInt32(asynUser *pasynUser, epicsInt32 value)
     /* Do callbacks so higher layers see any changes */
     callParamCallbacks();
 
-    /* Report any errors */
-    if (status)
-        asynPrint(pasynUser, ASYN_TRACE_ERROR,
-              "%s:writeInt32 error, status=%d function=%d, value=%d\n",
-              driverName, status, function, value);
-    else
-        asynPrint(pasynUser, ASYN_TRACEIO_DRIVER,
-              "%s:writeInt32: function=%d, value=%d\n",
-              driverName, function, value);
+    if (function != AravisConnection)
```

Comment:
It's always running (1 hz as I recall) and thus makes it hard to use TRACEIO_DRIVER to follow what's happening w/ a specific command.  Plus it doesn't tell us anything useful for diagnostics to print the same trace msg over and over.